### PR TITLE
Munaim

### PR DESCRIPTION
**Pull request by AbdulMunaim0099**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,25 +6,315 @@
...
+                                    <div class="text-[#ffffffb2] text-sm self-center">Last entry 7:34 AM</div>
+                                    <div class="border-2 border-[#b1b1b1b2] rounded-full w-10 h-10 flex justify-center items-center">
+                                        <svg class="w-7 h-7" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><g id="SVGRepo_bgCarrier" stroke-width="0"></g><g id="SVGRepo_tracerCarrier" stroke-linecap="round" stroke-linejoin="round"></g><g id="SVGRepo_iconCarrier"> <path d="M4 12H20M12 4V20" stroke="#ffffff" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"></path> </g></svg>
+                                    </div>
+                                </div>
+
+                            </div>
+
+                        </div>
+                        <div class="w-[55%] h-full  ">
```
Comment (muhammadilyas28) [inline index.html:472]: dont ue h-full fix it and make h-screen

**Review by muhammadilyas28 — CHANGES_REQUESTED** [review]
a bit little

**Review by imtiaz70 — APPROVED** [review]
Done! Shabash Abdul Munaim

File: index.html (hunk 1 above)
Comment (AbdulMunaim0099) [reply index.html:472]: h-screen Done Check karo dehan saye

**Review by muhammadilyas28 — APPROVED** [review]
Shabash Munaim Beta

**Pull request merged**